### DISCUSSION TITLE
EthUsdPriceFeed problem

### DISCUSSION THREAD

**Discussion by Krishduttm** [smartcontractkit/full-blockchain-solidity-course-py#992] [Q&A] [answered]
In lesson 7 when we test -

` def test_get_entranceFee():

    lottery = deploy_lottery()
    expected_entrance_fee = Web3.toWei(0.025, "ether")
    entrance_fee = lottery.getEntranceFee()

    assert expected_entrance_fee == entrance_fee`

why is the 'entrance_fee' value coming as 25 * 10^15, and not the current price? Which should be around (50/3000) = 0.016 = 16 * 10^15?

**Comment by cromewar** [comment, marked as answer]
Hello @Krishduttm 
This is because we setup manually the ranges of prices  to $50 and is not getting the actual price, we are suppose at first the price is $2000, but don't worry about this just continue the lesson and eventually you'll get the correct price values using chainlink.

**Reply by Krishduttm** [reply]
Okay got it! I have another doubt. I see that while deploying on testnet , only accounts[0] is used. If i added more accounts how will i get the contract to use and enter the lottery through all those accounts?

**Reply by cromewar** [reply]
This could be done in different ways:

1. declaring a lot of accounts like (`account1 = accounts[0], account2 = accounts[0]`)
2. Use a loop which uses the different accounts with a index.
3. Almost any other thing you could think about.

The key part here is you must remember each transaction (deploy, fund, function call, etc) should be done by an account with the identifier `{"from": account_name}` if you don't use this, brownie will use the default account it has or the one you used to deploy the contract.
